react/components/FooterVtexLogo.js

```diff
@@ -4,10 +4,12 @@ import PropTypes from 'prop-types'
 import withImage from './withImage'
 import footer from '../footer.css'
 
+const isPlatformGC = account => account.indexOf('gc_') === 0 || account.indexOf('gc-') === 0
```

Comment:
Use [Ramda's includes](https://ramdajs.com/docs/#includes) instead

Reply:
This don't make sense to me due to Ramda perform a indexOf too, but ok...
https://github.com/ramda/ramda/blob/v0.26.1/source/internal/_includes.js
https://github.com/ramda/ramda/blob/v0.26.1/source/internal/_indexOf.js